src/skibidi_orm/migration_engine/adapters/sqlite3_typing.py

```diff
@@ -0,0 +1,19 @@
+from typing import Literal
+from skibidi_orm.migration_engine.adapters.base_adapter import (
+    BaseColumn,
+    Relation,
+    BaseTable,
+)
+
+from skibidi_orm.migration_engine.operations.constraints import Constraint
+
+
+class SQLite3Typing:
+
+    DataTypes = Literal["TEXT", "INTEGER", "REAL", "BLOB", "NULL"]
+    Constraints = Constraint
+    Column = BaseColumn[DataTypes]
+    Table = BaseTable[Column]
+    Relation = Relation
+    tables: list[Table] = []
+    relations: list[Relation] = []
```

Comment:
I don't know if creating whole new class for types is necessary, when we have this data in BaseColumn.
For example tables, relations, Relation, Table, Column, Constraint will all be most likely the same in every DB engine, and we would just need to provide one or two new type aliases